.github/CODEOWNERS

```
@@ -335,4 +335,4 @@ Modules/_xxtestfuzz/          @ammaraskar
 # Remote debugging
 Python/remote_debug.h         @pablogsal
 Python/remote_debugging.c     @pablogsal
-Modules/_remote_debugging_module.c @pablogsal @ambv @1st1
\ No newline at end of file
+Modules/_remote_debugging_module.c @pablogsal @ambv @1st1
```

you could optionally simplify with:
```suggestion
Python/remote_debug*          @pablogsal
Modules/_remote_debugging*    @pablogsal @ambv @1st1
```

---

I prefer to have them excplicitly stated as that pattern has already caused people to be added to new future files that shouldn't be like the jit stuff